util/formula_templater/formula_test.go

```diff
@@ -13,7 +13,10 @@ func TestPrintOSSFormula(t *testing.T) {
 	version := "1.13.2"
 	config := "./config.hcl"
 	buf := new(bytes.Buffer)
-	expect := `class Consul < Formula
+	expect := `# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+class Consul < Formula
```

Comment:
Head's up this has changed on main and is now in testdata. Though I don't see any reason to put licensing bits on the testdata.